.gitignore

```diff
@@ -4,4 +4,4 @@
 /telegraf.exe
 /telegraf.gz
 /vendor
-.DS_Store
+.DS_Store
```

Comment:
This is unrelated, please revert. We really want to have a newline there. :-)

Reply:
Reverted my bad!